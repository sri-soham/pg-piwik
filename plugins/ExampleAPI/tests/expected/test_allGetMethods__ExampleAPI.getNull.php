a:2:{s:6:"result";s:7:"success";s:7:"message";s:2:"ok";}