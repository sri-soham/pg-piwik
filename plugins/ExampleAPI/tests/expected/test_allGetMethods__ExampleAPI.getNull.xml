<?xml version="1.0" encoding="utf-8" ?>
<result>
	<success message="ok" />
</result>